src/tests/roc_rtcp/test_communicator.cpp

```diff
@@ -3818,97 +3832,6 @@
     }
 }
 
-// Same, but there is a persistent clock drift between sender and receiver
-TEST(communicator, rtt_clock_drift) {
```

Comment:
If this test doesn't work due to smoothing of RTT, then we can disable smoothing specifically in this test by setting RttConfig fields (in some comment above I suggested to move smoothing settings to config).